scripts/release/homebrew/docker/formula_generate.py

```diff
@@ -134,7 +134,8 @@ def update_formula() -> str:
 
     # update python version
     text = re.sub('depends_on "python@.*"', f'depends_on "python@{PYTHON_VERSION}"', text, 1)
-    text = re.sub(r'virtualenv_create\(libexec, "python.*"', f'virtualenv_create(libexec, "python{PYTHON_VERSION}"', text, 1)  # pylint: disable=line-too-long
+    venv_str = f'venv = virtualenv_create(libexec, "python{PYTHON_VERSION}", system_site_packages: false)'
```

Comment:
Is `system_site_packages` a new parameter?

Reply:
This is an existing param.

In 3.11, the venv statement is `venv = virtualenv_create(libexec, "python3.11", system_site_packages: false, without_pip: false)`

Ref: https://github.com/Homebrew/homebrew-core/blob/bf1b3470f443c375da31e99d3b366b1ed8e8fb62/Formula/a/azure-cli.rb#L722

Reply:
Haha, I see this param is suggested by you: https://github.com/Homebrew/brew/issues/11885, and introduced in https://github.com/Azure/azure-cli/pull/19663